src/components/reportPage/ReportPage.jsx

```diff
@@ -51,6 +81,33 @@ const ReportPage = (props) => {
     setSearchString(e.target.value);
   };
 
+  const handleMenuChange = (e) => {
+    getValue(Array.isArray(e) ? e.map((x) => x.value) : []);
+  };
+
+  const checkImageFilterType = (imageResult) => {
+    const found = displayValue.some((value) => imageResult["bug_type"].includes(value));
+
+    if (
+      displayValue.length === 0 ||
+      (displayValue.includes("1") && imageResult["bug_type"].length === 0) ||
+      found
+    ) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Could be shortened to `return displayValue.length === 0 ||
      (displayValue.includes("1") && imageResult["bug_type"].length === 0) ||
      found`